### ISSUE TITLE
sim2h_server newrelic error

### ISSUE THREAD

**Issue by Connoropolous** [holochain/holochain-rust#2118]
Running `sim2h_server` where the version is v0.0.43-alpha3 results in an error:
```bash
$ sim2h_server -V
2020-02-17 17:07:44.425 -0500 (18408 4743669) error: failed to connect to the daemon using a timeout of 0 ms at the path /tmp/.newrelic.sock
sim2h_server 0.0.43-alpha3
```

**Comment by freesig** [comment]
This isn't so much a bug as just a report that it hasn't connected to the newrelic server. But I don't think it should emit this error unless the env var is set and then it can't connect.

**Comment by freesig** [comment]
Also it should probably be a warn not an error

**Comment by Connoropolous** [comment]
True... it's just strange that the default logging has this line show up when you run `sim2h_server` and nothing else...
```
2020-02-17 17:07:44.425 -0500 (18408 4743669) error: failed to connect to the daemon using a timeout of 0 ms at the path /tmp/.newrelic.sock
```
That's the whole log unless you change the logging level

**Comment by freesig** [comment]
Yeh it happens before logging is enabled
